Tests/scripts/configure_tests.py

```diff
@@ -646,14 +685,130 @@ def get_test_from_conf(branch_name):
     return tests
 
 
-def get_test_list(files_string, branch_name):
+def is_test_runnable(test_id, id_set, conf, server_version):
```

Comment:
i want to get notified when my test is skipped, lets fail the build on this

Reply:
I'm not sure this is what we're looking for...
@yuvalbenshalom what do you think?